modules/porous_flow/include/materials/PorousFlowMaterialCapillaryPressureVG.h

```diff
@@ -0,0 +1,57 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+
+#ifndef POROUSFLOWMATERIALCAPILLARYPRESSUREVG_H
+#define POROUSFLOWMATERIALCAPILLARYPRESSUREVG_H
+
+#include "PorousFlowMaterialCapillaryPressureBase.h"
+
+//Forward Declarations
+class PorousFlowMaterialCapillaryPressureVG;
+
+template<>
+InputParameters validParams<PorousFlowMaterialCapillaryPressureVG>();
+
+/**
+ * van Genuchten capillary pressure for multiphase flow in porous media.
+ *
+ * Based on van Genuchten, M. Th., A closed for equation for
+ * predicting the hydraulic conductivity of unsaturated soils,
+ * Soil Sci. Soc., 44, 892-898 (1980).
+ */
+class PorousFlowMaterialCapillaryPressureVG : public PorousFlowMaterialCapillaryPressureBase
+{
+public:
+  PorousFlowMaterialCapillaryPressureVG(const InputParameters & parameters);
+
+protected:
+
+  virtual void computeQpProperties();
+
+  /// Capillary pressure as a function of phase saturation
+  Real capillaryPressure(Real saturation) const;
+  /// Derivative of capillary pressure wrt phase saturation
+  Real dCapillaryPressure(Real saturation) const;
+  /// Second derivative of capillary pressure wrt phase saturation
+  Real d2CapillaryPressure(Real saturation) const;
+  /// Effective saturation
+  Real effectiveSaturation(Real saturation) const;
```

Comment:
I agree that the doc comment merely repeats the method name here, however, i would like to keep the doc.  Particularly with derivatives - there are so many variables here that i want to make sure the reader knows exactly what is differentiated with respect to what.